app/templates/index.html

```diff
@@ -3,24 +3,23 @@
   <head>
     <meta charset="utf-8">
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1<% if (includeBootstrap && !legacyBootstrap) { %>, shrink-to-fit=no<% } %>">
+    <meta name="viewport" content="width=device-width, initial-scale=1<% if (includeBootstrap) { %>, shrink-to-fit=no<% } %>">
     <title><%= appname %></title>
 
     <link rel="apple-touch-icon" href="apple-touch-icon.png">
     <!-- Place favicon.ico in the root directory -->
 
-    <!-- build:css styles/vendor.css -->
-    <!-- bower:css -->
-    <!-- endbower -->
-    <!-- endbuild -->
-
     <!-- build:css styles/main.css -->
     <link rel="stylesheet" href="styles/main.css">
     <!-- endbuild -->
     <% if (includeModernizr) { %>
-    <!-- build:js scripts/vendor/modernizr.js -->
-    <script src="/bower_components/modernizr/modernizr.js"></script>
-    <!-- endbuild --><% } %>
+    <script src="scripts/modernizr.js"></script>
```

Comment:
Is this going to be uglified in production? If not, I think we should keep the `build:js` comments.

Reply:
God idea! Let me change that now ^^

Reply:
This is what I meant:

```html
<!-- build:js scripts/vendor/modernizr.js -->
<script src="scripts/modernizr.js"></script>
<!-- endbuild -->
```

This way it will be compressed in production.

Reply:
Now should be fine here https://github.com/yeoman/generator-webapp/pull/737/commits/f9ccfa817a58e43497423f16d1acb7eab2e4e013